src/Integration.Vsix/SonarLintTracker/IssueTracker.cs

```diff
@@ -52,17 +55,17 @@ internal class IssueTracker : ITagger<IErrorTag>, IDisposable
 
         internal IssuesSnapshot Snapshot { get; set; }
 
-        internal IssueTracker(TaggerProvider provider, ITextBuffer buffer, ITextDocument document, string sqLanguage)
+        internal IssueTracker(_DTE dte, TaggerProvider provider, ITextBuffer buffer, ITextDocument document, List<IContentType> contentTypes)
```

Comment:
I would use `IList<IContentType>` instead, e.g. the interface, not the concrete type (in C# the convention is to name interfaces with first letter `I`).